CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+ - `MirrorNodeCloseClientIntegrationTest`
```

Comment:
I don't think we should document test changes in the changelog. It's assumed fixes would have a corresponding test.